src/kanata/mod.rs

```diff
@@ -556,6 +559,13 @@ impl Kanata {
                 } else {
                     mmsv.distance = mmas.max_distance;
                 }
+                for &modifier in &self.move_mouse_speed_modifiers {
+                    mmsv.distance = u16::max(
```

Comment:
I haven't tested, but on inspection, modifying this state seems incorrect in the case of non-accelerated mouse movement. This will multiply (and persist) on every tick.